gitlab.go

```diff
@@ -187,9 +187,6 @@ func (c *Client) SetBaseURL(urlStr string) error {
 		return err
 	}
 
-	// Set the encoded opaque data
-	c.baseURL.Opaque = c.baseURL.Path
-
```

Comment:
I'm totally with you concerning the use of Go 1.5, but I don't think it's up to me or you to decide when people upgrade their software.

Next to that this change doesn't fix anything, so functionally there is no reason to change the current implementation. And with the current implementation the Go version you use doesn't matter. So changing the code doesn't fix or improve anything but it does limit the usability of the package in other packages...

Think it's clear that we should not change this now :wink: